articles/how-to-control-dark-mode-in-react-native-using-redux/index.md

```diff
@@ -0,0 +1,445 @@
+- [React Navigation](https://medium.com/@sultanbutt820/react-navigation-v5-whats-new-d8048f76a0de#:~:text=Apr%2026%C2%B73%20min%20read,command%20in%20the%20project%20terminal.)
+- [React Native Paper UI library](https://reactnativepaper.com/#:~:text=React%20Native%20Paper%20is%20a,in%20all%20major%20use%2Dcases.)
+- [React Redux](https://www.valentinog.com/blog/redux/)
+- [Using Redux mechanism in React Native](https://enappd.com/blog/redux-in-react-native-app/92/)
+
+### STEP 1: Setup simple screens with React Native paper
+Our first step is to set up simple screens using the components from the react-native-paper UI library.
+
+In order to do that, we need to create a directory named **./screens** and inside the directory, we need to create two new files called `Home.js` and `Setting.js`.
+
+In the Home.js file, we need to import the components from the react-native-paper package as directed in the code snippet below:
+
+```
+import {
+    Avatar,
+    Button,
+    Card,
+    Title,
+    Paragraph,
+    List,
+    Headline,
+  } from 'react-native-paper';
+```
+
+Next, we are going to add the card layout to the screen using the `Card` component as shown in the code snippet below:
+
+```
+const Home = () => {
+   return (
+       <View>
+            <Headline style={{ marginLeft: 23 }}>Lastest Post</Headline>
+           <Card
+               style={{
+                   shadowOffset: { width: 5, height: 5 },
+                   width: '90%',
+                   borderRadius: 12,
+                   alignSelf: 'center',
+                   marginBottom: 10,
+               }}>
+               <Card.Content>
+                   <Title>Blog post</Title>
+                   <Card.Cover
+                       style={{
+                           width: '100%',
+                           height: 190,
+                           alignSelf: 'center',
+                       }}
+                       source={{
+                           uri:
+                               'https://images.unsplash.com/photo-1573921470445-8d99c48c879f?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1050&q=80',
+                       }}
+                   />
+                   <Paragraph>What is Lorem Ipsum?
+Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged.
+i</Paragraph>
+               </Card.Content>
+           </Card>
+
+       </View>
+   )
+}
+export default Home
+```
+
+Here, we have used the Headline component to display the title. To create a card like component in the UI, we used the Card component with its subsidiary modules. Inside the Navigation.js file, we need to import the React Navigation components.
```

Comment:
I changed this and deleted a sentence because I believe it was repeating?

Reply:
yep, seem I over-explain thank.